celery_healthcheck/server.py

```diff
@@ -1,40 +1,68 @@
+import logging
 import threading
 
 import uvicorn
 from celery import bootsteps
-from fastapi import FastAPI
+from celery.worker import WorkController
+from fastapi import FastAPI, status
 from fastapi.responses import JSONResponse
 
 app = FastAPI()
+logger = logging.getLogger("celery.ext.healthcheck")
+
+HEALTHCHECK_DEFAULT_PORT = 9000
+HEALTHCHECK_DEFAULT_PING_TIMEOUT = 2.0
 
 
 class HealthCheckServer(bootsteps.StartStopStep):
-    def __init__(self, worker, **kwargs):
-        self.worker = worker
+    def __init__(self, parent: WorkController, **kwargs):
```

Comment:
`parent` in this case references the specific worker instance/container that the healthcheck server is running beside, correct?